app/javascript/elm/src/Main.elm

```diff
@@ -103,6 +106,9 @@ init flags url key =
 
                 _ ->
                     Mobile
+
+        sensors =
+            Sensor.decodeSensors flags.sensors
```

Comment:
just to leave a trace you were talking about changing this to `decode` and refactoring the code around this

Reply:
Thanks!